### DISCUSSION TITLE
Store validation loss during fitting

### DISCUSSION THREAD

**Discussion by kargintima** [Nixtla/neuralforecast#742] [Q&A]
I have looked through the documentation already, but didn't find a way to store validation loss during fitting a model.
So I could choose hyperparameters, initiate NeuralForecast() instance and make it fit. And in the end I want to be able to plot training curve - loss(epoch).
Is it possible?
Of course, if I add 'verbose' option:
`NeuralForecast.fit(df, val_size=val_size, verbose=True)`
I can parse output and extract validation loss after each 100 epochs. But it is a little bit weird workaround.
